app/src/main/java/com/kms/cura/view/activity/PatientBasicSettingsActivity.java

```diff
@@ -136,23 +163,87 @@ private String getFormatedCityString() {
         String formatedCity = null;
         try {
             formatedCity = new String(entity.getLocation().getBytes("ISO-8859-1"), "UTF-8");
-        } catch (UnsupportedEncodingException e) {
+        } catch (Exception e) {
```

Comment:
Should not change from UnsupportedEncodingException  to general Exception like this